.github/workflows/codeql-analysis.yml

```diff
@@ -0,0 +1,53 @@
+name: "CodeQL"
+
+on:
+  push:
+    branches: [master, ]
+  pull_request:
+    # The branches below must be a subset of the branches above
+  schedule:
+    - cron: '0 15 * * 0'
```

Comment:
So we want to run this daily? What happens if it fails overnight, someone gets an email?

Reply:
We can set up appropriate alerting in the security settings to send out those emails.